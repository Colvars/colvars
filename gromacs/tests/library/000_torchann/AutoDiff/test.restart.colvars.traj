# step         nn_0                  r_nn_0                v_nn_0                vr_nn_0               ft_nn_0               fa_nn_0               
          20    6.03556982902622e+01  6.30066233938420e+01  0.00000000000000e+00 -1.57069174779280e-02 -1.35629728273979e-01  1.28862467369851e-01 
          21    6.03418730609519e+01  6.29895748263702e+01 -1.38252293103704e-02 -1.70485674718298e-02 -1.28849565908995e-01  1.35321025859088e-01 
          22    6.03500651663749e+01  6.29712516783066e+01  8.19210542306337e-03 -1.83231480635335e-02 -1.20347246616352e-01  1.41177824309496e-01 
          23    6.03776954677088e+01  6.29517380546391e+01  2.76303013338932e-02 -1.95136236675453e-02 -1.10411041100995e-01  1.46410313292816e-01 
          24    6.04218817028334e+01  6.29311322443654e+01  4.41862351245632e-02 -2.06058102737192e-02 -9.93460489849742e-02  1.51010773338691e-01 
          25    6.04795395481504e+01  6.29095437024464e+01  5.76578453170029e-02 -2.15885419189940e-02 -8.74655140267484e-02  1.54984615298090e-01 
          26    6.05475148928954e+01  6.28870899511971e+01  6.79753447450366e-02 -2.24537512492343e-02 -7.50790242983713e-02  1.58348673529888e-01 
          27    6.06227130017898e+01  6.28638935178400e+01  7.51981088943339e-02 -2.31964333571611e-02 -6.24811473859655e-02  1.61129378133531e-01 
          28    6.07021734326250e+01  6.28400790206404e+01  7.94604308352689e-02 -2.38144971996251e-02 -4.99455721361506e-02  1.63360847683030e-01 
          29    6.07830229161676e+01  6.28157704615754e+01  8.08494835425648e-02 -2.43085590650041e-02 -3.77313203107576e-02  1.65083108791173e-01 
          30    6.08625950742698e+01  6.27910886640890e+01  7.95721581022093e-02 -2.46817974863226e-02 -2.60718228835410e-02  1.66340819468198e-01 
          31    6.09383357545880e+01  6.27661489639920e+01  7.57406803181979e-02 -2.49397000970397e-02 -1.51855578527981e-02  1.67181846012829e-01 
          32    6.10077490628933e+01  6.27410590482759e+01  6.94133083052790e-02 -2.50899157161364e-02 -5.28208747645170e-03  1.67656394695729e-01 
          33    6.10684689018857e+01  6.27159168821223e+01  6.07198389924051e-02 -2.51421661535646e-02  3.44473243180521e-03  1.67816457952591e-01 
          34    6.11182499551973e+01  6.26908087912803e+01  4.97810533116620e-02 -2.51080908420164e-02  1.08153721812520e-02  1.67715142292832e-01 
          35    6.11549958276929e+01  6.26658076861574e+01  3.67458724955512e-02 -2.50011051228405e-02  1.66670764098739e-02  1.67406131659082e-01 
          36    6.11768684735809e+01  6.26409714518431e+01  2.18726458880241e-02 -2.48362343142926e-02  2.08644126352624e-02  1.66943157314363e-01 
          37    6.11824133132800e+01  6.26163416084097e+01  5.54483969911956e-03 -2.46298434334651e-02  2.33111447024902e-02  1.66379254270167e-01 
          38    6.11704780647566e+01  6.25919423589437e+01 -1.19352485234003e-02 -2.43992494660071e-02  2.39412710875282e-02  1.65765803093786e-01 
          39    6.11403005341279e+01  6.25677799366587e+01 -3.01775306287553e-02 -2.41624222849949e-02  2.27272436574744e-02  1.65151924347952e-01 
          40    6.10912701794129e+01  6.25438423323889e+01 -4.90303547150148e-02 -2.39376042697817e-02  1.96554698952972e-02  1.64583743256515e-01 
