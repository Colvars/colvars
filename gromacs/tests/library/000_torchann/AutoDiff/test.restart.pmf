# 1
# -1.82500000000000e+02  5.00000000000000e+00         72  1

 -1.80000000000000e+02   4.11075017905465e-01
 -1.75000000000000e+02   4.02510955032434e-01
 -1.70000000000000e+02   3.93946892159404e-01
 -1.65000000000000e+02   3.85382829286373e-01
 -1.60000000000000e+02   3.76818766413343e-01
 -1.55000000000000e+02   3.68254703540312e-01
 -1.50000000000000e+02   3.59690640667282e-01
 -1.45000000000000e+02   3.51126577794251e-01
 -1.40000000000000e+02   3.42562514921221e-01
 -1.35000000000000e+02   3.33998452048190e-01
 -1.30000000000000e+02   3.25434389175160e-01
 -1.25000000000000e+02   3.16870326302129e-01
 -1.20000000000000e+02   3.08306263429099e-01
 -1.15000000000000e+02   2.99742200556068e-01
 -1.10000000000000e+02   2.91178137683038e-01
 -1.05000000000000e+02   2.82614074810007e-01
 -1.00000000000000e+02   2.74050011936977e-01
 -9.50000000000000e+01   2.65485949063946e-01
 -9.00000000000000e+01   2.56921886190916e-01
 -8.50000000000000e+01   2.48357823317885e-01
 -8.00000000000000e+01   2.39793760444855e-01
 -7.50000000000000e+01   2.31229697571824e-01
 -7.00000000000000e+01   2.22665634698794e-01
 -6.50000000000000e+01   2.14101571825763e-01
 -6.00000000000000e+01   2.05537508952732e-01
 -5.50000000000000e+01   1.96973446079702e-01
 -5.00000000000000e+01   1.88409383206671e-01
 -4.50000000000000e+01   1.79845320333641e-01
 -4.00000000000000e+01   1.71281257460610e-01
 -3.50000000000000e+01   1.62717194587580e-01
 -3.00000000000000e+01   1.54153131714549e-01
 -2.50000000000000e+01   1.45589068841519e-01
 -2.00000000000000e+01   1.37025005968488e-01
 -1.50000000000000e+01   1.28460943095458e-01
 -1.00000000000000e+01   1.19896880222427e-01
 -5.00000000000000e+00   1.11332817349397e-01
  0.00000000000000e+00   1.02768754476366e-01
  5.00000000000000e+00   9.42046916033357e-02
  1.00000000000000e+01   8.56406287303052e-02
  1.50000000000000e+01   7.70765658572747e-02
  2.00000000000000e+01   6.85125029842442e-02
  2.50000000000000e+01   5.99484401112136e-02
  3.00000000000000e+01   5.13843772381831e-02
  3.50000000000000e+01   4.28203143651526e-02
  4.00000000000000e+01   3.42562514921221e-02
  4.50000000000000e+01   2.56921886190916e-02
  5.00000000000000e+01   1.71281257460610e-02
  5.50000000000000e+01   8.56406287303052e-03
  6.00000000000000e+01   0.00000000000000e+00
  6.50000000000000e+01   6.08048463985166e-01
  7.00000000000000e+01   5.99484401112136e-01
  7.50000000000000e+01   5.90920338239105e-01
  8.00000000000000e+01   5.82356275366075e-01
  8.50000000000000e+01   5.73792212493044e-01
  9.00000000000000e+01   5.65228149620014e-01
  9.50000000000000e+01   5.56664086746983e-01
  1.00000000000000e+02   5.48100023873953e-01
  1.05000000000000e+02   5.39535961000922e-01
  1.10000000000000e+02   5.30971898127892e-01
  1.15000000000000e+02   5.22407835254861e-01
  1.20000000000000e+02   5.13843772381831e-01
  1.25000000000000e+02   5.05279709508800e-01
  1.30000000000000e+02   4.96715646635770e-01
  1.35000000000000e+02   4.88151583762739e-01
  1.40000000000000e+02   4.79587520889709e-01
  1.45000000000000e+02   4.71023458016678e-01
  1.50000000000000e+02   4.62459395143647e-01
  1.55000000000000e+02   4.53895332270617e-01
  1.60000000000000e+02   4.45331269397586e-01
  1.65000000000000e+02   4.36767206524556e-01
  1.70000000000000e+02   4.28203143651525e-01
  1.75000000000000e+02   4.19639080778495e-01
