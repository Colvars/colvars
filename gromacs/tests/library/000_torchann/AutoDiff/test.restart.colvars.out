colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: Summary of compile-time features available in this build:
colvars:   - SMP parallelism: enabled (num. threads = 2)
colvars:   - Multiple replicas: not available
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: not available
colvars: ----------------------------------------------------------------------
colvars: Start colvars Initialization.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars:   prot_N (10 atoms)
colvars:   prot_CA (10 atoms)
colvars:   prot_C (10 atoms)
colvars:   prot_O (10 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # defaultInputStateFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "nn_0"
colvars:   Initializing a new "torchANN" component.
colvars:   Warning: cannot cite unknown feature "torchANN colvar component (derived from linearCombination)"
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     # modelFile = "identity.pt"
colvars:     torch model loaded.
colvars:     # m_output_index = 0
colvars:     # doubleInputTensor = on
colvars:     Input dimension of model: 1
colvars:     Model's dtype: kFloat64.
colvars:     Evaluating model with zero tensor succeeded.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 5
colvars:   # lowerBoundary = -180
colvars:   # upperBoundary = 180
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = on
colvars:   Enabling the extended Lagrangian term for colvar "nn_0".
colvars:   # extendedTemp = 300
colvars:   # extendedFluctuation = 5
colvars:   Computed extended system force constant: 0.0997735 [E]/U^2
colvars:   # extendedTimeConstant = 200
colvars:   Computed fictitious mass: 101.092 [E]/(U/fs)^2   (U: colvar unit)
colvars:   # outputEnergy = off [default]
colvars:   # extendedLangevinDamping = 0
colvars:   # reflectingLowerBoundary = off [default]
colvars:   # reflectingUpperBoundary = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = on
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { nn_0 }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 0
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # CZARestimator = on [default]
colvars:   # integrate = on [default]
colvars:   # UIestimator = off [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "nn_0":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "nn_0":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: Re-initialized atom group for variable "nn_0":0/2. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: Re-initialized atom group for variable "nn_0":0/3. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - GROMACS engine:
colvars:   Abraham2015 https://doi.org/10.1016/j.softx.2015.06.001
colvars: 
colvars: - Colvars module:
colvars: - Optimal rotation via flexible fitting:
colvars: - dihedral colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - ABF colvar bias implementation:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - CZAR eABF estimator:
colvars: - eABF implementation:
colvars:   Lesage2017 https://doi.org/10.1021/acs.jpcb.6b10055
colvars: 
colvars: - Colvars-GROMACS interface:
colvars: - linearCombination colvar component:
colvars:   n/a 
colvars: 
colvars: The final output state file will be "test.restart.part0002.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Loading state from unformatted memory.
colvars:   Restarting collective variable "nn_0" from value: 60.3557
colvars:   Restarted abf bias "abf1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: End colvars Initialization.
colvars: 
colvars:   extended (x,v): 63.0066,-0.0157069
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars:   extended (x,v): 62.9896,-0.0170486
colvars:   extended (x,v): 62.9713,-0.0183231
colvars:   extended (x,v): 62.9517,-0.0195136
colvars:   extended (x,v): 62.9311,-0.0206058
colvars:   extended (x,v): 62.9095,-0.0215885
colvars:   extended (x,v): 62.8871,-0.0224538
colvars:   extended (x,v): 62.8639,-0.0231964
colvars:   extended (x,v): 62.8401,-0.0238145
colvars:   extended (x,v): 62.8158,-0.0243086
colvars:   extended (x,v): 62.7911,-0.0246818
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
colvars:   extended (x,v): 62.7661,-0.0249397
colvars:   extended (x,v): 62.7411,-0.0250899
colvars:   extended (x,v): 62.7159,-0.0251422
colvars:   extended (x,v): 62.6908,-0.0251081
colvars:   extended (x,v): 62.6658,-0.0250011
colvars:   extended (x,v): 62.641,-0.0248362
colvars:   extended (x,v): 62.6163,-0.0246298
colvars:   extended (x,v): 62.5919,-0.0243992
colvars:   extended (x,v): 62.5678,-0.0241624
colvars:   extended (x,v): 62.5438,-0.0239376
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.part0002.colvars.traj".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
colvars: Saving collective variables state to "test.restart.part0002.colvars.state".
